### PR TITLE
Progress bars for key generation

### PR DESCRIPTION
**Pull request by CarlBeek**
When generating many keys, it can be frustrating not knowing how long it takes to generate your keys. This PR helps mitigate this frustration by offering progress bars:
```text
Creating your keys:               [####################################]  16/16          
Creating your keystores:          [####################################]  16/16          
Creating your depositdata:        [####################################]  16/16          
Verifying your keystores:         [####################################]  16/16          
Verifying your deposits:          [####################################]  16/16          

Success!
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,11 +111,11 @@ You can use `--help` flag to see all arguments.
...
-Creating your keys.
-Saving your keystore(s).
-Creating your deposit(s).
-Verifying your keystore(s).
-Verifying your deposit(s).
+Creating your keys:               [####################################]  <N>/<N>
+Creating your keystores:          [####################################]  <N>/<N>
+Creating your depositdata:        [####################################]  <N>/<N>
+Verifying your keystores:         [####################################]  <N>/<N>
+Verifying your deposits:          [####################################]  <N>/<N>
```
Comment (hwwhww) [inline README.md:118]: If my nitpicking comments got included, need to modify the README.md a bit too.

File: eth2deposit/utils/validation.py [hunk 2]
```diff
@@ -26,7 +27,9 @@ def verify_deposit_data_json(filefolder: str) -> bool:
     """
     with open(filefolder, 'r') as f:
         deposit_json = json.load(f)
-        return all([validate_deposit(deposit) for deposit in deposit_json])
+        with click.progressbar(deposit_json, label='Verifying your deposits:\t',
```
Comment (hwwhww) [inline eth2deposit/utils/validation.py:30]:
```suggestion
        with click.progressbar(deposit_json, label='Verifying your deposit(s):\t',
```

**Review by hwwhww — APPROVED** [review]
Nice feature! 🦏 

Just some nitpicking.

**Comment by CarlBeek** [comment]
I decided to remove the "(s)" everywhere, because I don't expect there to be too many single deposits and even if there are, I don't think it is the end of the world so assume the plural. IMO "(s)" just looks ugly.

**Pull request merged**